client/hosting-overview/components/plan-card.tsx

```diff
@@ -36,87 +37,81 @@ const PlanCard: FC = () => {
 	return (
 		<>
 			<QuerySitePlans siteId={ site?.ID } />
-			<Card className="hosting-overview__card">
-				<div>
-					<div className="hosting-overview__plan-card-header">
-						<h3 className="hosting-overview__plan-card-title">{ planName }</h3>
+			<Card className={ classNames( 'hosting-overview__card', 'hosting-overview__plan' ) }>
```

Comment:
We don't need `classnames` for this:
```suggestion
			<Card className="hosting-overview__card hosting-overview__plan">
```